### PR TITLE
Fill in details for CVE-2021-25737

### PR DESCRIPTION
**Pull request by cjcullen**
@joelsmith @tallclair PTAL

### CODE REVIEW

File: 2021/25xxx/CVE-2021-25737.json [hunk 1]
```diff
@@ -1,18 +1,110 @@
...
+    "affects": {
+        "vendor": {
+            "vendor_data": [
+                {
+                    "product": {
+                        "product_data": [
+                            {
+                                "product_name": "Kubernetes",
+                                "version": {
+                                    "version_data": [
```
Comment (tallclair) [inline 2021/25xxx/CVE-2021-25737.json:17]: We still need to figure out how we want to convey the version information for unsupported versions.... I don't have a good answer right now though. In the past we've listed out every minor version, but that's annoying and doesn't scale well.

@joelsmith do you have any thoughts on this?

**Review by tallclair — APPROVED** [review]
lgtm, just the question about the versions.

As an aside, we should right a linter / validation script. It wouldn't need to be too complex, just make sure the fields we care about are present, and the fields that are constant (e.g. `assigner`) are set to the right values.

**Comment by tallclair** [comment]
> As an aside, we should right a linter / validation script. It wouldn't need to be too complex, just make sure the fields we care about are present, and the fields that are constant (e.g. assigner) are set to the right values.

https://github.com/kubernetes-security/cvelist-public/pull/4

File: 2021/25xxx/CVE-2021-25737.json (hunk 1 above)
Comment (cjcullen) [reply 2021/25xxx/CVE-2021-25737.json:18]: Merging as is. We can revisit older version question later.

**Pull request merged**